### PR TITLE
improve dev experience

### PR DESCRIPTION
**Pull request by dannyrb**
fixes https://github.com/OHIF/Viewers/issues/857
fixes https://github.com/OHIF/Viewers/issues/704
fixes https://github.com/OHIF/Viewers/issues/903

### CODE REVIEW

**Comment by dannyrb** [comment]
image

File: .webpack/webpack.dev.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+module.exports = {
+  entry: {
+    examples: ENTRY_EXAMPLES,
+  },
+  devtool: 'source-map',
+  output: {
+    path: OUT_PATH,
+    filename: '[name].bundle.[hash].js',
+    library: 'cornerstoneViewport',
```
Comment (swederik) [inline .webpack/webpack.dev.js]: If you are changing the name, you also need to change the package.json.

File: examples/ExamplePageGrid.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+                <label htmlFor="image-id-stack">Image ID Stack:</label>
+                <select
+                  defaultValue={1}
+                  onChange={evt => {
+                    const selectedStack =
+                      parseInt(evt.target.value) === 1 ? stack1 : stack2;
+
+                    this.setState({
+                      imageIds: selectedStack,
+                      imageIdIndex: 0,
```
[the remark this answers is not in the document]
Comment (dannyrb) [reply examples/ExamplePageGrid.js]: Eh. I can. Right now, all 4 viewport's props are being provided by the same state. I try to demonstrate the "how we track the active viewport" and "using the known active viewport to do something special (styling)", but I didn't want to mess with maintaining the state separately for each viewport for a quick example.

File: examples/ExamplePageEscapeHatch.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                name: 'Wwwc',
+                mode: 'active',
+                modeOptions: { mouseButtonMask: 1 },
+              },
+            ]}
+            imageIds={[
+              'dicomweb://s3.amazonaws.com/lury/PTCTStudy/1.3.6.1.4.1.25403.52237031786.3872.20100510032220.11.dcm',
+              'dicomweb://s3.amazonaws.com/lury/PTCTStudy/1.3.6.1.4.1.25403.52237031786.3872.20100510032220.12.dcm',
+            ]}
+            onElementEnabled={elementEnabledEvt => {
```
[the remark this answers is not in the document]
Comment (dannyrb) [reply examples/ExamplePageEscapeHatch.js]: FWIW, if you've grabbed the enabled element, your tool is likely already registering listeners outside the lifecycle of this component

File: .webpack/webpack.dev.js (hunk 1 above)
Comment (dannyrb) [reply .webpack/webpack.dev.js]: Actually just the dev server's config. Still using rollup on this one for publishing packages. Dev server's just handy for.. Developing ^_^

**Comment by alvarosan** [comment]
Just realizing this is already merged!  Does this mean that  [viewers#1007](https://github.com/OHIF/Viewers/issues/1007) and [viewers#1006](https://github.com/OHIF/Viewers/issues/1006) are now supported? As @ivan-aksamentov already mentioned, we could definitely contribute upstream if there are any bits an pieces left in order to achieve this but, we would need some guidance.

**Pull request merged**